java/src/main/c++/jni_spark_vw.cc

```diff
@@ -135,9 +135,69 @@ JNIEXPORT jobject JNICALL Java_org_vowpalwabbit_spark_VowpalWabbitNative_getArgu
   jstring args = env->NewStringUTF(serializer.str().c_str());
 
   jclass clazz = env->FindClass("org/vowpalwabbit/spark/VowpalWabbitArguments");
-  jmethodID ctor = env->GetMethodID(clazz, "<init>", "(IILjava/lang/String;)V");
+  jmethodID ctor = env->GetMethodID(clazz, "<init>", "(IILjava/lang/String;DD)V");

```

Comment:
What does this DD mean?

Reply:
int, int, string, double, double in the signature (2 doubles added)

Reply:
it's the constructor signature.

Reply:
Got it, thanks